<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="grafoviWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>KONCENTRACIJE</string>
       </attribute>
       <layout class="QVBoxLayout" name="koncPanelLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="zsWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>ZERO / SPAN</string>
       </attribute>
       <layout class="QVBoxLayout" name="zsPanelLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
      <widget class="QWidget" name="visednevni">
       <attribute name="title">
        <string>PREGLED AGREGIRANIH</string>
       </attribute>
       <layout class="QVBoxLayout" name="visednevniLayout"/>
      </widget>
      <widget class="QWidget" name="komentari">
       <attribute name="title">
        <string>KOMENTARI</string>
       </attribute>
       <layout class="QVBoxLayout" name="komentariLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1159</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_REST">
    <property name="title">
     <string>&amp;REST</string>
    </property>
    <addaction name="action_log_in"/>
    <addaction name="action_log_out"/>
    <addaction name="separator"/>
    <addaction name="action_reconnect"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="action_stil_grafova"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_info_aplikacija"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_REST"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Kontrolni prozor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_log_in"/>
   <addaction name="action_log_out"/>
   <addaction name="action_reconnect"/>
   <addaction name="separator"/>
   <addaction name="action_stil_grafova"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_log_in">
   <property name="text">
    <string>&amp;Log in</string>
   </property>
  </action>
  <action name="action_log_out">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Log out</string>
   </property>
  </action>
  <action name="action_reconnect">
   <property name="text">
    <string>&amp;Reconnect</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_stil_grafova">
   <property name="text">
    <string>&amp;Stil grafova</string>
   </property>
   <property name="toolTip">
    <string>Promjena stila grafova.</string>
   </property>
  </action>
  <action name="action_info_aplikacija">
   <property name="text">
    <string>&amp;O Aplikaciji</string>
   </property>
   <property name="toolTip">
    <string>informacije</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
